src/agents/ipp_group.jl

```diff
@@ -5483,7 +5577,7 @@ function solve_agent_problem_ipp_cap(
         optimize!(MPPDCMER_lower)
     end
 
-    objective_value(MPPDCMER_lower)
+    # objective_value(MPPDCMER_lower)
```

Comment:
Why is this commented out? (What does calling `objective_value` do?)

Reply:
`objective_value` will get the objective value of the problem. If we leave it here and the problem is infeasible, then the program will error out (stop) because there's no objective value for infeasible problems.